### DISCUSSION TITLE
Is there any way that core support to copy files from master server to distributed server, or directly create files on distributed server?

### DISCUSSION THREAD

**Discussion by www-uu** [coreemu/core#871] [Q&A]
**Question:**
I need to copy files from master server to distributed server, or directly create files on distributed server.
Is there any way that core support this?

**Guess:**

1. I found "services are provided as utilities"  include ftp, in doc: [https://coreemu.github.io/core/services/utility.html#overview](https://coreemu.github.io/core/services/utility.html#overview), so maybe can use ftp by core?
2.  I found "the distributed servers must be configured to allow passwordless root login over SSH", in doc: [https://coreemu.github.io/core/distributed.html](https://coreemu.github.io/core/distributed.html), so maybe can use scp / paramiko by core?

**Comment by bharnden** [comment]
Services on distributed nodes will create and copy files over to distributed servers.
